Source/Mosa.DeviceDriver/ISA/PIT.cs

```diff
@@ -104,5 +117,16 @@ public uint GetTickCount()
 		{
 			return tickCount;
 		}
+
+		/// <summary>
+		/// Waits for a specific time, in milliseconds.
+		/// </summary>
+		public void Wait(uint ms)
```

Comment:
I think this method should be external to PIT.

Reply:
Where should it be then? It uses properties of PIT so I think it should be in PIT.